seaborn/utils.py

```diff
@@ -889,3 +891,13 @@ def _disable_autolayout():
 def _version_predates(lib: ModuleType, version: str) -> bool:
     """Helper function for checking version compatibility."""
     return Version(lib.__version__) < Version(version)
+
+
+def try_convert_to_pandas(data: object | None) -> pd.DataFrame:
```

Comment:
Why `object | None` and not `Any`?

Reply:
just because `Any` turns off the type checker so I tend to avoid using it unless I have to, whereas as `object` prevents me from making assumptions about what properties the variable might have